codejam3-pokemon/Program.cs

```diff
@@ -28,80 +31,80 @@ public static async Task Main(string[] args)
                                 `'                            '-._|                                                                                       
                                                                            ";
 
-        Console.WriteLine(title);
-
-        Console.WriteLine("Welcome to the Pokedex!");
-
-        //Console.WriteLine("Enter the name of a Pokemon to see its stats!");
-
-        //var p = Console.ReadLine();
-
-        bool quit = false;
+            Console.WriteLine(title);
+            Console.WriteLine("Welcome to the Pokedex!");
+        await startGame();
+    }
 
-        while (!quit)
+    public static async Task startGame()
+    {
+        Console.WriteLine("Enter the name of a Pokemon to see its stats! Enter quit to quit.");
+        var p = Console.ReadLine()?.ToLower();
+        bool hasArt = false; 
+        if (p == "quit")
         {
-            Console.WriteLine("Enter the name of a Pokemon to see its stats! Enter quit to quit.");
-            var p = Console.ReadLine()?.ToLower();
-
-            if (p == "quit")
-            {
-                quit = true;
-                break;
-            }
-            else
+            quitGame();
+        }
+        else if (! await isRealPoke(p)) 
+        {
+                await startGame();
+        } else
+          {
+                hasArt = PrintPoke(p.ToLower());
+          }
+        bool quit2 = false;
+        while (!quit2)
+        {
+            if (!hasArt) { Console.ForegroundColor = ConsoleColor.Blue; }
+            Console.WriteLine("---------------------------------------------");
```

Comment:
I like adding the lines as dividers to space out the text and make it easier to see